app/src/main/java/org/brightmindenrichment/street_care/ui/community/CommunityEventFragment.kt

```diff
@@ -766,6 +769,33 @@ class CommunityEventFragment : Fragment(), AdapterView.OnItemSelectedListener {
 
     }
 
+
+    private fun filterEventsBySkill(skill: String, isPastEvents: Boolean) {
+
+        // Refresh the event list with the filtered query
+        refreshEvents(
+            eventDataAdapter,
+            this@CommunityEventFragment.resources,
+            if(isPastEvents) {
+                getQueryToFilterPastEventsByType(skill, isPastEvents)
```

Comment:
Could this simplified as the isPartEvents argument is delegated 

```
        refreshEvents(
            eventDataAdapter = eventDataAdapter,
            resources = this@CommunityEventFragment.resources,
            query = getQueryToFilterPastEventsByType(skill, isPastEvents),
            inputText = userInputText
        )
        searchEvents(
            eventDataAdapter = eventDataAdapter,
            resources = this@CommunityEventFragment.resources,
            query = getQueryToFilterFutureEventsByType(skill, isPastEvents),
        )

Reply:
Addressed